validation/validation_test.go

```diff
@@ -127,9 +120,9 @@ func TestEmailDomainValidationCorrectlyValidatesAllowedDomains(t *testing.T) {
 		emailResponse := IsAllowedEmailDomain(email, allowedDomains)
 		So(emailResponse, ShouldBeFalse)
 
-		newAllowedDomains := append(allowedDomains, "@gmail.com")
+		allowedDomains = append(allowedDomains, "@gmail.com")
```

Comment:
This is now modifying a variable outside the scope of this `Convey`. Is this really what you want?